Sources/AdminPanel/routes.swift

```diff
@@ -38,16 +38,28 @@ public struct AdminPanelEndpoints {
     }
 }
 
+public struct AdminPanelMiddlewares: Service {
```

Comment:
I guess this doesn't need to be a `Service` since we're going with this approach?